lib/tiled_component.dart

```diff
@@ -1,12 +1,12 @@
-import 'package:flame/components/component.dart';
+import 'package:flame/components.dart';
 import 'package:tiled/tiled.dart';
 
 import 'dart:ui';
 
 import './tiled.dart';
 
 class TiledComponent extends Component {
-  Tiled _tiled;
+  Tiled? _tiled;
```

Comment:
```suggestion
  late final Tiled _tiled;
```

Reply:
And also remove all `_tiled!` exclamation marks.

Reply:
OK!